app/src/main/java/com/andryoga/safebox/ui/view/chooseMasterPswrd/ChooseMasterPswrdFragment.kt

```diff
@@ -37,12 +44,51 @@ class ChooseMasterPswrdFragment : Fragment() {
         binding.viewModel = viewModel
         binding.lifecycleOwner = this
 
+        binding.hintText.addTextChangedListener {
+            val similarLength = longestCommonSubstring(
+                binding.hintText.text.toString().toLowerCase(),
+                binding.pswrdText.text.toString().toLowerCase()
+            )
+
+            if (similarLength >= Constants.maxSimilarLength) {
+                binding.hint.isErrorEnabled = true
+                binding.hint.error = getString(R.string.hint_error)
+                binding.saveBtn.isEnabled = false
+            } else {
+                binding.hint.isErrorEnabled = false
+                binding.hint.error = null
+                binding.saveBtn.isEnabled = true
```

Comment:
open app
just enter hint..lets say "a"
save button is enabled (without any password)...

Reply:
yeah I just checked that I will try to fix it